docs/learn-scams.md

```diff
@@ -0,0 +1,62 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
+### Types of Scams
+
+- Personal messages over Telegram, Twitter, and other social media
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM private keys/seeds/etc., posing as a member of our team
+- Scammers will take official videos, add text to it so that it looks like an official video
+- Many scammers will create nearly perfect imitations of sites - triple-check the URL
+- People offering to help you stake or get rewards
+- People answering questions you asked publicly in a private chat
+- Scammers will often make it seem like the 'deal' is only available for a limited time. Do not be
+  tricked by this, it is always better to confirm than to risk losing everything.
+
+### Admin will never contact you over Telegram, nor any other Social Media
+
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
+personally message you. Our social media's are posted on our website and any new social media's will
+be announced by our team. We will never offer to sell you DOTs at a discount, air-drop "rewards", or
+message you privately to help with a problem you posted publicly.
+
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, URL, etc. If something seems fishy, that's because it likely
+is. Never enter any personal data after clicking on a link, always type it in yourself. Feel free to
```

Comment:
Not clear what "always type it in yourself" refers to, looks like Personal data. Rephrase or separate sentences.